examples/ReorderExample.js

```diff
@@ -92,29 +92,27 @@ class ReorderExample extends React.Component {
         isColumnReordering={false}
         width={1000}
         height={500}
+        columnsCount={7}
+        getColumn={(i) => {
+          return {
+            allowCellsRecycling: true,
+            columnKey: this.state.columnOrder[i],
+            key: i,
+            header: (
+              <Plugins.ReorderCell
+                onColumnReorderStart={onColumnReorderStart}
+                onColumnReorderEnd={onColumnReorderEndCallback}
+              >
+                {columnTitles[this.state.columnOrder[i]]}
+              </Plugins.ReorderCell>
+            ),
+            cell: <TextCell data={dataList} />,
+            fixed: fixedColumns.indexOf(this.state.columnOrder[i]) !== -1,
```

Comment:
Does this work as expected? Doesn't the new API have a constraint that fixed columns should appear before scrollable columns.

Reply:
Yeah, but fixedColumns array has first and second columns, that why its working fine.